docs/pipelines/tasks/deploy/azure-vmss-deployment.md

```diff
@@ -46,6 +46,22 @@ Use this task to deploy a virtual machine scale set image.
 
 </table>
 
+
+## Troubleshooting
+
+### Error: Permission denied
+
+This issue happens when we try to run a custom script but the script isn't executable. 
```

Comment:
Can you give a more detailed error message. Just `Permission denied` is very generic.